src/test/java/org/kohsuke/github/GHProjectCardTest.java

```diff
@@ -55,6 +55,8 @@ public void testCreateCardFromIssue() throws IOException {
             GHIssue issue = repo.createIssue("new-issue").body("With body").create();
             GHProjectCard card = column.createCard(issue);
             assertThat(card.getContentUrl(), equalTo(issue.getUrl()));
+            assertThat(card.getContent(), notNullValue());
+            assertThat(card.getContent().getRepository(), notNullValue());
```

Comment:
This needs to assert some meaningful information, such as Id and/or description.  Just `notNullValue()` is not a sufficient test.

Reply:
Agreed, this was just to get the right API calls being made.